stac_mjx/main.py

```diff
@@ -56,7 +56,7 @@ def run_stac(
     start_time = time.time()
 
     # Getting paths
-    fit_path = base_path / cfg.stac.fit_path
+    fit_offsets_path = base_path / cfg.stac.fit_offsets_path
```

Comment:
Both api_usage.ipynb and run_stac.py are broken because `fit_offsets_path` isn't in our configs. Did you forget to commit some changes?